spec/index.html

```diff
@@ -1110,6 +1148,27 @@ <h4>Patterns of RDF entailment (Informative)</h4>
             _:x rdf:type xsd:boolean .
             _:x rdf:type xsd:integer .
           </pre>
+          
+         <p><span style="background-color: Cyan;color: Black">
+         The semantics conditions for triple terms give the following RDF entailment pattern: 
+         </span></p>
+      
+         <table style="background-color: Cyan;color: Black">
+         <caption style="background-color: Cyan;color: Black">RDF entailment patterns with triple terms.</caption>
+           <tbody style="background-color: Cyan;color: Black">
+             <tr style="background-color: Cyan;color: Black">
+               <th > </th>
+               <th ><strong>if <em>S</em> contains</strong></th>
+               <th ><strong>then <em>S</em> RDF entails</strong></th>
+             </tr>
+             <tr>
+               <td class="othertable" style="background-color: Cyan;color: Black"><dfn>rdfREIF1</dfn></td>
+               <td class="othertable" style="background-color: Cyan;color: Black">sss aaa ttt <code>.</code><br/> for ttt a triple term</td>
+               <td class="othertable" style="background-color: Cyan;color: Black">aaa <code>rdf:type</code> <code>rdf:ReificationProperty .</code><br/>sss aaa _:nnn <code>.</code><br/>_:nnn <code>rdf:type</code><code> rdf:TripleTerm </code><code>.</code></td>
```

Comment:
In my opinion, this should be restricted to `rdf:reifies` only.
Other ReificationProperties may or may not have range `rdf:TripleTerm`.

ReificationProperty was proposed as a mere "flag" to allow people to detect errors (IF they consider this as an error). With this inference rule, it goes beyond flagging, and has other important consequences.

Reply:
I tendo to agree here.

Reply:
actually, I misread this inference rule. It does *not* imply that all ReificationProperties would all have range `rdf:TripleTerm`, and I have no issue with it.